scripts/entity/tactical/enemies/rf_bandit_baron.nut

```diff
@@ -156,23 +158,8 @@ this.rf_bandit_baron <- ::inherit("scripts/entity/tactical/human", {
 		local r = ::Math.rand(1, 100);
 		if (r <= 25)
 		{
-			local weapon = ::MSU.Class.WeightedContainer([
-				[1, "scripts/items/weapons/named/named_axe"],
-				[1, "scripts/items/weapons/named/named_cleaver"],
-				[1, "scripts/items/weapons/named/named_spear"],
-				[1, "scripts/items/weapons/named/named_sword"],
-				[1, "scripts/items/weapons/named/named_hammer"],
-				[1, "scripts/items/weapons/named/named_mace"],
-
-				[1, "scripts/items/weapons/named/named_rf_longsword"],
-				[1, "scripts/items/weapons/named/named_rf_kriegsmesser"],
-				[1, "scripts/items/weapons/named/named_rf_swordstaff"],
-				[1, "scripts/items/weapons/named/named_two_handed_flail"],
-				[1, "scripts/items/weapons/named/named_two_handed_mace"],
-				[1, "scripts/items/weapons/named/named_greatsword"]
-			]).roll();
-
-			this.m.Items.equip(::new(weapon));
+			local weapon = ::Reforged.ItemTable.NamedNorthernWeaponMelee.roll()
```

Comment:
This will allow a bandit baron to spawn with a named spetum ... which feels awkward to me because that isn't a weapon worthy of the topmost tier bandit. It also feels out of place thematically considering the other weapon choices. I looked at weapon values and thought maybe we can exclude spetum that way but 2h flail is the same value i.e. 2800 so that won't work.

Maybe exclude the spetum manually here.